### PR TITLE
Add LIMIT to topic_authors.sparql

### LINKED ISSUE #1842: In Topic aspect, add LIMIT to ```Authors publishing about the topic``` panel
**What kind of panel are you referring to?**
The  ```Authors publishing about the topic``` panel, e.g. as per https://scholia.toolforge.org/topic/Q210861#authors .

**What kind of information should the panel provide, and which of the visualization options (e.g. table, bubble chart, map) should it use?**

It works fine in most cases, just when there is too much information, it might fail, so adding a LIMIT would be helpful.

Screenshot from 2022-02-16 13-52-29

**Which Wikidata entries would be good candidates to explore this?**

Heavily curated topics, such as the ones on [this list](https://w.wiki/4qhS)

### PR DESCRIPTION
**Pull request by Daniel-Mietchen**
Fixes #1842 

Just added a LIMIT to the SPARQL query.

### CODE REVIEW

**Comment by fnielsen** [comment]
Shouldn't the `LIMIT` be before the labeling, to avoid spending time on labeling items that are not shown?

**Comment by Daniel-Mietchen** [comment]
I tested this for a number of the highly curated topics, e.g. [schizophrenia](https://scholia.toolforge.org/topic/Q41112). While the LIMIT I chose is 200 (as in many other panels), the number of results we get is sometimes more than that (e.g. 203 right now for Schizophrenia), due to people having multiple ORCIDs. I think this should be fine, but we could make it always 200 by adding ```LIMIT 200``` at the bottom again.

Screenshot from 2022-02-16 14-28-10

File: scholia/app/templates/topic_authors.sparql [hunk 1]
```diff
@@ -1,13 +1,16 @@
+# tool: scholia
 #defaultView:Table
 SELECT ?count ?author ?authorLabel (COALESCE(?orcid_, CONCAT("orcid-search/quick-search/?searchQuery=", ?authorLabel)) AS ?orcid) 
 WITH {
-  SELECT (count(?work) as ?count) ?author WHERE {
+  SELECT DISTINCT ?author (count(?work) as ?count) WHERE {
```
Comment (fnielsen) [inline scholia/app/templates/topic_authors.sparql:5]: Is the DISTINCT necessary when we have the GROUP BY?

**Comment by Daniel-Mietchen** [comment]
Adding a pointer to 
- #1807 

which serves to keep an overview of things we worked on during the hackathon.

**Pull request merged**